maven/bnd-resolver-maven-plugin/pom.xml

```diff
@@ -84,7 +88,7 @@
                       <jsse.enableSNIExtension>false</jsse.enableSNIExtension>
                     </properties>
                     <goals>
-                        <goal>bnd-resolver:resolve</goal>
+                        <goal>package</goal>
```

Comment:
No because the invoker is invoking a reactor... not indivdual projects of which there are more than one.